sway-lsp/tests/lib.rs

```diff
@@ -1638,7 +1638,7 @@ fn hover_docs_for_functions_vscode() {
         req_uri: &uri,
         req_line: 20,
         req_char: 14,
-        documentation: vec!["```sway\npub fn bar(p: Point) -> Point\n```\n---\n A function declaration with struct as a function parameter\n\n---\nGo to [Point](command:sway.goToLocation?%5B%7B%22range%22%3A%7B%22end%22%3A%7B%22character%22%3A1%2C%22line%22%3A5%7D%2C%22start%22%3A%7B%22character%22%3A0%2C%22line%22%3A2%7D%7D%2C%22uri%22%3A%22file","sway%2Fsway-lsp%2Ftests%2Ffixtures%2Ftokens%2Ffunctions%2Fsrc%2Fmain.sw%22%7D%5D \"functions::Point\")"],
+        documentation: vec!["```sway\npub fn bar(p: Point) -> Point\n```\n---\n A function declaration with struct as a function parameter\n\n---\nGo to [Point](command:sway.goToLocation?%5B%7B%22range%22%3A%7B%22end%22%3A%7B%22character%22%3A1%2C%22line%22%3A5%7D%2C%22start%22%3A%7B%22character%22%3A0%2C%22line%22%3A2%7D%7D%2C%22uri%22%3A%22file","sway-lsp%2Ftests%2Ffixtures%2Ftokens%2Ffunctions%2Fsrc%2Fmain.sw%22%7D%5D \"functions::Point\")"],
```

Comment:
It looks like the `sway/` is getting stripped off the beginning of the file path, which is used for the hover links. I don't think this should change.

Reply:
I'm not sure `sway` belongs there since that's the name of `repo-root/..` as I only found this out because I do git checkouts via worktrees:

    git worktrees add path/to/sway/<issue-number>/ origin/master

And so when the tests ran, I was getting ".../sway/**_6179_**/sway-lsp/..." which is why I think the `sway/` prefix is hardcoded but shouldn't be there.

Reply:
Fair enough. I tested it and the links seem to be working fine